check_404.py

```
@@ -0,0 +1,61 @@
+
+import requests
+from bs4 import BeautifulSoup
+from urllib.parse import urljoin, urlparse
+
+BASE_URL = "https://yohi.github.io/docs-manager/"
+
+def get_links(url):
+    try:
+        response = requests.get(url)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)